server/src/middleware/errorHandler.ts

```diff
@@ -42,6 +42,15 @@ export function errorHandler(err: unknown, req: Request, res: Response, next: Ne
   if (err instanceof ApiError) {
     res.status(err.statusCode);
 
+    // If multiple errors not undefined, return that in errors property.
```

Comment:
We could have a `getErrors(): Array<string>` method in `ApiError`, which returns either a single error message in an array of length 1, or the `multipleErrors` property, so we don't need to check whether `multipleErrors` is defined here.

Reply:
Good point, added getter to the ApiError class and updated error handler.